NuRadioReco/modules/channelCWNotchFilter.py

```diff
@@ -0,0 +1,227 @@
+import numpy as np
+from scipy import signal
+from NuRadioReco.utilities import units
+from NuRadioReco.utilities import fft
+
+"""
+Contains function to filter continuous wave out of the signal
+functions should work on a per event basis to comply with the iteration methods used in readRNOGData
```

Comment:
In general all modules work like this. You do not need to mention that here. And I would remove the reference to `readRNOGData`. This module can also be used for other data/experiments